samples/todos/src/main/java/com/toggl/komposable/sample/todos/TodosApp.kt

```diff
@@ -41,7 +42,7 @@ val todosAppReducer: Reducer<TodosState, TodosAction> = TodosReducer().forEachLi
     mapToElementList = { state -> state.todos },
     mapToParentAction = { action, index -> TodosAction.Todo(index, action) },
     mapToParentState = { state, todosMap -> state.copy(todos = todosMap) },
-)
+).run { DebugReducer(this, logger = { Log.d("debug", it) }) }
```

Comment:
You probably just didn't get to it yet, but I'll just mention that we wanted some nice [extension method](https://github.com/pointfreeco/swift-composable-architecture/blob/f660a699d3cb45841bfac3dc7d76957d8ce348f4/Sources/ComposableArchitecture/Reducer/Reducers/DebugReducer.swift#L15) to do this and completely semi-hide the DebugReducer itself